lib/onc_certification_g10_test_kit/bulk_export_validation_tester.rb

```diff
@@ -197,11 +220,16 @@ def perform_bulk_export_validation
       end
 
       @resources_from_all_files = {}
+      @first_error = {}
+      @invalid_line_count = 0
       success_count = 0
+
       file_list.each do |file|
         success_count += check_file_request(file['url'])
       end
 
+      process_validation_errors(success_count)
```

Comment:
It's confusing to me that `success_count` is being passed as an argument to `process_validation_errors`, when it seems that inside `process_validation_errors` it represents the total number of resources. It may be that `success_count` made sense as a variable name before, but now that we're continuing on validation failures a name like `resource_count` would make more sense now.